public/locales/en/translation.json

```diff
@@ -222,6 +222,10 @@
         },
         "title": "Downloads"
     },
+    "emptyLibrary": {
+        "noGames": "Your library is empty. You can <1>log in</1> using a store or click <3></3> to add one manually.",
```

Comment:
these `<1>` and `<3>` are intentional, the `Trans` component replaces them with the tags from the children